Models/Entities/Receipt.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace PSP.Models.Entities
+{
+    public class Receipt
+    {
+        public int OrderId { get; set; }
```

Comment:
Missing relational property publid Order Order { get; set; } Will be hard to make a receipt with related products and services without it.